spec/features/merchant/dashboard_to_do_list_spec.rb

```diff
@@ -0,0 +1,136 @@
+      expect(page).to have_link(@pull_toy2.name)
+      expect(page).to have_link(@dog_bone2.name)
+
+      click_link @tire2.name
+    end
+
+    expect(page).to have_current_path("/merchant/items/#{@tire2.id}/edit")
+    expect(find_field(:Image).value).to have_content('https://static.wixstatic.com/media/d8d60b_6ff8d8667db1462492d681839d85054c~mv2.png/v1/fill/w_900,h_900,al_c,q_90/file.jpg')
+  end
+
+  it "I see a message saying that no items need their images updated" do
+    click_link 'Logout'
+    bike_shop = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+    employee = bike_shop.users.create!(
+        name: "Steve",
+        address:"123 Main St.",
+        city: "Fort Collins",
+        state: "GA",
+        zip: "66666",
+        email: "chunky_merchant@example.com",
+        password: "123password",
+        role: 1
+      )
+    
+    visit "/login"
+    fill_in :Email, with: employee.email
+    fill_in :Password, with: employee.password
+    click_button "Login"
+    
+    within "#to-do-list" do
+      expect(page).to have_content("No images to update!")
+    end
+  end
+
+  it "I see a section of the to do list that lists the amount of money I have in unfulfilled orders" do
+    user1 = User.create(name: 'Steve Meyers', address: '555 Free St.', city: 'Plano', state: 'TX', zip: '88992', email: "user1@example.com", password: "user1")
+    order1 = user1.orders.create(name: 'Steve Meyers', address: '555 Free St.', city: 'Plano', state: 'TX', zip: '88992', status: 0)
+    order2 = user1.orders.create(name: 'Steve Meyers', address: '555 Free St.', city: 'Plano', state: 'TX', zip: '88992', status: 0)
+    order3 = user1.orders.create(name: 'Steve Meyers', address: '555 Free St.', city: 'Plano', state: 'TX', zip: '88992', status: 0)
+    ItemOrder.create(item: @tire, order: order1, price: @tire.price, quantity: 2)
+    ItemOrder.create(item: @dog_bone, order: order2, price: @dog_bone.price, quantity: 4)
+    ItemOrder.create(item: @pull_toy, order: order3, price: @pull_toy.price, quantity: 4)
+
+    visit "/merchant/dashboard"
+
+    expect(page).to have_content("Unfulfilled Orders:")
+    expect(page).to have_content("You have 3 unfulfilled orders worth $324.00")
+  end
+
+  it "I see that all my items have been fulfilled" do
+    click_link 'Logout'
+    bike_shop = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+    employee = bike_shop.users.create!(
+        name: "Steve",
+        address:"123 Main St.",
+        city: "Fort Collins",
+        state: "GA",
+        zip: "66666",
+        email: "chunky_merchant@example.com",
+        password: "123password",
+        role: 1
+      )
+    
+    visit "/login"
+    fill_in :Email, with: employee.email
+    fill_in :Password, with: employee.password
+    click_button "Login"
+
+    expect(page).to have_content("All orders fulfilled!")
+  end
```

Comment:
some of these tests could probably be more robust by addressing sad paths more directly, could be a spot for me to improve later